src/business-logic/services/inews-gateway-connection-implementation.ts

```diff
@@ -0,0 +1,64 @@
+import { Device } from '../../model/entities/device'
+import { DeviceConnection } from './interfaces/inewsgateway-device-connection'
+
+export class INewsGatewayDeviceConnection implements DeviceConnection {
+  private readonly device: Device
+  private isConnectingOrDisconnecting: boolean = false
+
+  constructor(_device: Device) {
+    this.device = _device
```

Comment:
The linting is configured to allow unused variables if they are prefixed with `_`, so this and other variables that have intended use should have the prefix removed. It won't class with the class member :)